### PR TITLE
Twitter Integration and Layout 1 Landing Page Improvement 

### PR DESCRIPTION
**Pull request by melvinalex7**
# Description

A summary of the change

- Twitter component for Layout 1 Light and Dark themes added.
- Recent Publications added to Landing Page.

# Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Refactoring
- [ ] Documentation
- [x] Improvement

# Solution description

Describe your code changes in detail for reviewers.
Explain the technical solution you have provided and how it fixes the issue.

# Tests

All unit tests pass?

Builds successfully?
Yes
Do all SA checks pass?

Attach screenshot of the expected result (e.g. UI of the website/ postman api call result/ database changes)
image

Provide steps in detail for reviewers to reproduce the expected result of the code changes.

# Relevant document/ link (if any)

Any related documents could help reviewers to review this PR

# Risk

Identity the risk level (low, medium, high) and indicate which component(s) will be affected if bugs exist.
Low

### CODE REVIEW

**Review by rsal0007 — CHANGES_REQUESTED** [review]
Looks good, but can we make the text formatting consistent? Basically change the way 'news' and 'recent publications' is written to how 'welcome to monteam' is written?

Also add some space between the final line of the text and the twitter + publications section? Would make the outlook cleaner.

**Comment by melvinalex7** [comment]
> Looks good, but can we make the text formatting consistent? Basically change the way 'news' and 'recent publications' is written to how 'welcome to monteam' is written?
> 
> Also add some space between the final line of the text and the news + publications section? Would make the outlook cleaner.

- The reason it had a different text style was that it was like the title of the landing page, whereas the rest were titles for page sections, which I thought required a different style than the title. However, I changed it as requested.
-  Resolved the padding issue as well.

**Comment by rsal0007** [comment]
> > Looks good, but can we make the text formatting consistent? Basically change the way 'news' and 'recent publications' is written to how 'welcome to monteam' is written?
> > Also add some space between the final line of the text and the news + publications section? Would make the outlook cleaner.
> 
> * The reason it had a different text style was that it was like the title of the landing page, whereas the rest were titles for page sections, which I thought required a different style than the title. However, I changed it as requested.
> * Resolved the padding issue as well.

Padding looks good now!

About the text, I meant making the section titles also in lowercase and bold, but I get that there should be a difference. Can we try something like this:
image


This way, it's distinct yet consistent. We can do this for every page (there are some inconsistencies in the titles anyway), happy to help out here! :)

**Review by jfan0010 — COMMENTED** [review]
Nice!

Just some personal opinion:
Since we're building layout 1 based on AllanLab, how about we remove the recent publication from homepage/landingpage and move them to publication page, like how allanlab does for their highlighted publication? 

AllanLab homepage:
image

AllanLab publication page:
image

File: scholly/base/src/layouts/layout1/components/landingPage/LandingPage.js [hunk 1]
```diff
@@ -1,14 +1,29 @@
...
   const homepageData = TEAM_HOMEPAGE;
-  const { teamName } = TEAM_INFO;
+  const themeOption = TEAM_SITE_METADATA.template.theme;
+  const { teamName, twitterHandle } = TEAM_INFO;
+  const pubs = TEAM_PUBLICATIONS;
+  const [height, setHeight] = useState(0);
+  const ref = useRef(null);
+
+  useEffect(() => {
+    setHeight((ref.current.clientHeight > 600) ? ref.current.clientHeight : 600);
```
Comment (aksh0001) [inline scholly/base/src/layouts/layout1/components/landingPage/LandingPage.js:25]: condense this logic by using a min function to ensure we're having 600 as the min

File: scholly/base/src/layouts/layout1/components/twitter/TwitterFeed.js [hunk 2]
```diff
@@ -5,10 +5,20 @@ import React from 'react';
...
-  </div>
-);
+const TwitterFeed = ({ linkedHandle, themeOption, twitterHeight }) => {
+  let borderColour = '#000000';
+  if (themeOption === 'dark') {
+    borderColour = '#ffffff';
+  }
+  console.log(twitterHeight);
+  return (
+    <div className="twitter-feed">
```
Comment (aksh0001) [inline scholly/base/src/layouts/layout1/components/twitter/TwitterFeed.js:15]: SA warnings

File: scholly/base/src/layouts/layout1/components/landingPage/LandingPage.js (hunk 1 above)
Comment (melvinalex7) [reply scholly/base/src/layouts/layout1/components/landingPage/LandingPage.js]: I think we should use the max function to ensure that 600 remains the minimum height for the twitter column.

Comment (aksh0001) [reply scholly/base/src/layouts/layout1/components/landingPage/LandingPage.js]: sgtm

File: scholly/base/src/global/fakeData.js [hunk 3]
```diff
@@ -2,8 +2,8 @@ export const FAKE_TEAM_SITE_METADATA = {
   pages: ['PUBLICATIONS', 'TEAM', 'ACHIEVEMENTS'],
   publicationOptions: { groupBy: 'Category', sortBy: 'Year' },
   template: {
-    layout: '2',
-    theme: 'light',
+    layout: '1',
```
Comment (jfan0010) [inline scholly/base/src/global/fakeData.js:5]: I think we can keep this file unchanged, as what Akshay said in other PR: https://github.com/Researchify/Researchify/pull/272#discussion_r725558292

File: scholly/base/src/global/fakeData.js [hunk 4]
```diff
@@ -156,7 +156,7 @@ export const FAKE_PUBLICATIONS = [
 ];
 
 export const FAKE_TEAM_INFO = {
-  twitterHandle: 'elonmusk',
+  twitterHandle: 'monashuni',
```
Comment (jfan0010) [inline scholly/base/src/global/fakeData.js:159]: same as above

**Pull request merged**